Doc/reference/ldap.rst

```diff
@@ -462,26 +462,23 @@ The module defines the following exceptions:
    are instead turned into exceptions, raised as soon an the error condition
    is detected.
 
-   The exceptions are accompanied by a dictionary possibly
-   containing an string value for the key :py:const:`desc`
-   (giving an English description of the error class)
-   and/or a string value for the key :py:const:`info`
-   (giving a string containing more information that the server may have sent).
-
-   A third possible field of this dictionary is :py:const:`matched` and
-   is set to a truncated form of the name provided or alias dereferenced
-   for the lowest entry (object or alias) that was matched.
-
-   The field :py:const:`msgid` is set in the dictionary where the
-   exception can be associated with an asynchronous request.
-   This can be used in asynchronous code where :py:meth:`result()` raises the
-   result of an operation as an exception. For example, this is the case for
-   :py:meth:`compare()`, always raises the boolean result as an exception
-   (:py:exc:`COMPARE_TRUE` or :py:exc:`COMPARE_FALSE`).
-
-   Most exceptions from protocol results also carry the :py:attr:`errnum`
-   attribute.
-
+   The exceptions are accompanied by a dictionary with additional information.
+   The following fields are set in the dictionary if the relevant information
+   is available:
+
+   * ``'desc'``: string giving an English description of the error class.
```

Comment:
It might be worth mentioning that the message is coming from libldap's ``ldap_err2string``.

The strings are not necessarily in English. libldap supports i18n through gettext and the error messages are marked for translation with ``N_("message")``.